swri_transform_util/src/transform_manager.cpp

```diff
@@ -363,4 +363,9 @@ namespace swri_transform_util
   {
     return GetTransform(target_frame, source_frame, ros::Time(0), timeout, transform);
   }
+
+  LocalXyWgs84UtilPtr &TransformManager::local_xy_util()
```

Comment:
This looks good to me, although to be consistent with the rest of the class, it would be better for this to be named using upper camel case, so `TransformManager::LocalXyUtil()`.